webapp/src/test/java/com/box/l10n/mojito/android/strings/XmlWriterTest.java

```diff
@@ -0,0 +1,164 @@
+package com.box.l10n.mojito.android.strings;
+
+import org.junit.Test;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.transform.TransformerException;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.box.l10n.mojito.android.strings.XmlHelper.*;
+import static org.junit.Assert.*;
+
+public class XmlWriterTest {
+
+    @Test
+    public void addStringTest() throws TransformerException, ParserConfigurationException {
+        XmlDocument xmlDocument = new XmlDocument()
```

Comment:
it is confusing that the class XmlDocument is directly instantiated but is an inner class of XmlHelper. if it is meant to be use directly as one of the main class it should be top level